main/webapp/modules/core/scripts/project/exporters.js

```diff
@@ -105,6 +105,18 @@ ExporterManager.prototype._initializeUI = function() {
   });
 };
 
+/**
+ * Adds a menu item to the ExporterManager menu.
+ * @public
+ * @param {object} item The menu item to add.
+ */
+ExporterManager.prototype.addMenuItem = function(item) {
+    // NOTE: while taking id, label, and click induvidually would
```

Comment:
```suggestion
    // NOTE: while taking id, label, and click individually would
```

Typo, but more importantly, at the risk of over engineering something trivial, perhaps we could address the comment by having two separate functions to make things explicit? e.g. `addMenuItem(id, label, click)` & `addMenuSpacer()` or something similar

Reply:
Yeah, I wanted to avoid new things and focus on documentation, but given that this is new no matter what, maybe I should go ahead and go straight for two new? From an extensions point of view the difference is minimal anyway.

Reply:
That would be my suggestion, unless @SoryRawyer or @wetneb have a different idea.